cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/util/FileUtil.java

```diff
@@ -29,4 +32,13 @@ public static String getFilenameFromUrl(String url) {
             throw new CseInvalidDataException(String.format("URL is invalid: %s", url), e);
         }
     }
+
+    public static String getDayOfWeekAndDifferenceBetweenBusinessHourAndPublicationHourFromInputCgm(String cgmFilename) {
```

Comment:
Method name is a bit confusing since it refers to a business concept
Perhaps rename it to describe more accurately what it _technically_ does, such as : 

- extractThreeDigitCodeBeforeExtension
- getThreeDigitCodeAfterDate
...